pmoves/Makefile

```
@@ -1867,6 +1867,110 @@ integration-contract-check-strict: ## Strict integration contract check (missing
 
+CF_PAGES_PROJECT ?= cataclysmstudios
+CF_PAGES_DIR ?= ../deploy/cataclysmstudios
+CF_PAGES_BRANCH ?= main
+
+CF_PAGES_PORT ?= 8788
+
+.PHONY: cf-pages-dev cf-pages-create cf-pages-deploy cf-pages-status
+cf-pages-dev: ## Serve cataclysmstudios locally via wrangler pages dev (http://localhost:CF_PAGES_PORT)
+	@echo "→ Previewing $(CF_PAGES_PROJECT) at http://localhost:$(CF_PAGES_PORT)"
+	npx wrangler pages dev "$(CF_PAGES_DIR)" --port $(CF_PAGES_PORT)
+cf-pages-create: ensure-env-shared ## Create Cloudflare Pages project (one-time bootstrap)
```

checkmake (0.3.2)

[warning] Target body for "cf-pages-create" exceeds allowed length of 5 lines (15).

(maxbodylength)